### PR TITLE
use the distroless image for development with a cluster

### PR DESCRIPTION
**Pull request by jgbernalp**
This PR adjusts the binary name in the distroless images. Also adjusts the metrics default port to avoid collisions while testing locally, fixes: #40

### CODE REVIEW

**Comment by Nexucis** [comment]
@jgbernalp to fix the CI, you need to update the file https://github.com/perses/perses-operator/blob/main/scripts/generate-goreleaser/.goreleaser.base.yaml#L5 to update the binary there

File: Dockerfile [hunk 1]
```diff
@@ -7,9 +7,9 @@ LABEL maintainer="The Perses Authors <perses-team@googlegroups.com>"
...
 USER nobody
 
-COPY --chown=nobody:nobody operator                      /bin/operator
+COPY --chown=nobody:nobody bin/manager                   /bin/manager
 COPY --chown=nobody:nobody LICENSE                       /LICENSE
 COPY --from=build-env --chown=nobody:nobody              /etc/mime.types /etc/mime.types
 
 EXPOSE     8080
-ENTRYPOINT [ "/bin/operator" ]
+ENTRYPOINT [ "/bin/manager" ]
```
[the remark this answers is not in the document]
Comment (jgbernalp) [reply Dockerfile:15]: The deployment was using a [command](https://github.com/perses/perses-operator/pull/41/files#diff-ce0d597a9e5f20a8fe61c3ba5e6185a6b90cfef2c00bb5b5ef3e97b15b79f480L63) to point to the `/manager` but I removed it so it uses the default image entrypoint . Which for consistency is now `/bin/manager`

File: distroless-debug.Dockerfile [hunk 2]
```diff
@@ -7,9 +7,9 @@ LABEL maintainer="The Perses Authors <perses-team@googlegroups.com>"
 
 USER nobody
 
-COPY --chown=nobody:nobody operator                      /bin/operator
+COPY --chown=nobody:nobody bin/manager                   /bin/operator
```
Comment (Nexucis) [inline distroless-debug.Dockerfile:10]:
```suggestion
COPY --chown=nobody:nobody bin/manager /bin/manager
```

File: .dockerignore [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # More info: https://docs.docker.com/engine/reference/builder/#dockerignore-file
 # Ignore build and test binaries.
 bin/
+!bin/manager
```
Comment (Nexucis) [inline .dockerignore:4]: why do you need to add this entry in `gitignore` ?

Comment (jgbernalp) [reply .dockerignore:4]: is on .dockerignore, otherwise the docker/podman will exclude the file when building the image

Comment (Nexucis) [reply .dockerignore:4]: ah yeah my bad

**Review by Nexucis — APPROVED** [review]

**Pull request merged**